vision-annotate-images/functions/src/index.ts

```diff
@@ -0,0 +1,29 @@
+import * as functions from "firebase-functions";
+import * as admin from "firebase-admin";
+import vision from "@google-cloud/vision";
+
+admin.initializeApp();
```

Comment:
nit: doesn't look like you actually use `firebase-admin` at all in this function?

Reply:
You're right. For some reason I thought it was required to do initializeApp. I've removed it.